app/views/state_file/questions/nj_eitc_qualifying_child/edit.html.erb

```diff
@@ -0,0 +1,42 @@
+<%
+    title = t(".title")
+    content_for :page_title, title
+%>
+<% content_for :card do %>
+  <h1 class="h2"><%= title %></h1>
+  <p class="p"><%= t(".instructions") %></p>
+  <%= form_with model: @form, url: { action: :update }, local: true, method: "put", builder: VitaMinFormBuilder do |f| %>
+    <div class="white-group">
+      <%= f.cfa_radio_set(:claimed_as_eitc_qualifying_child, 
+        label_text: t('.primary_eitc_qualifying_child_question', filing_year: @filing_year),
+        collection: [
+              { value: :yes, label: t("general.affirmative") },
+              { value: :no, label: t("general.negative") },
+        ]
+        ) %>
+    </div>
+    <% if current_intake.filing_status_mfj?%>
+      <div class="white-group">
+        <%= f.cfa_radio_set(:spouse_claimed_as_eitc_qualifying_child, 
+          label_text: t('.spouse_eitc_qualifying_child_question', filing_year: @filing_year),
+          collection: [
+                { value: :yes, label: t("general.affirmative") },
+                { value: :no, label: t("general.negative") },
+          ]
+          ) %>    
+      </div>
+    <% end %>
+
+    <div class="reveal">
+      <p><button class="reveal__button"><%= t('.helper_heading') %></button></p>
```

Comment:
[pebble] button doesn't need to be in a p tag

Reply:
good catch. i copied this from [here](https://github.com/codeforamerica/vita-min/blob/aa974becb8653ab12131a122fecc5477389d845a/app/views/state_file/questions/nj_homeowner_property_tax/edit.html.erb#L15) so this can be fixed in at least one other place in a followup

Reply:
https://github.com/newjersey/affordability-pm/issues/153 now links here